app/src/main/java/org/oppia/android/app/player/state/itemviewmodel/SelectionInteractionViewModel.kt

```diff
@@ -148,6 +153,18 @@ class SelectionInteractionViewModel private constructor(
     }
   }
 
+  private fun updateSelectionText() {
+    if (selectedItems.size < maxAllowableSelectionCount) {
+      selectedItemText.set("You may select more choices")
```

Comment:
This & the others should be strings that are loaded from the resources file, rather than hardcoded (so that they may be translated).

Reply:
Okay @BenHenning